glslc/README.asciidoc

```diff
@@ -185,13 +196,17 @@ preprocessing errors are written to standard error. If multiple input shader
 files are given, their preprocessed output are all written to standard output,
 in the order specified on the command line.
 
+glslc will do nothing for SPIR-V assembly files with this option.
```

Comment:
`-fshader-stage=` actually has effects on `.spvasm` files. It will treat `.spvasm` files as GLSL source code of the specified stage. I decided to use this behavior after `clang -x`. `clang -x some.s` will error out.